### ISSUE TITLE
Unexpected token error

### ISSUE THREAD

**Issue by Matodor** [wheatjs/vite-plugin-vue-type-imports#16]
After plugin installation i got this error on many pages (where defineProps is used):
`[plugin:vite-plugin-vue-type-imports] Unexpected token, expected "</>/<=/>=" (2:7)`

![image](https://user-images.githubusercontent.com/1778417/176917781-f8c12e1a-cbd1-4218-9738-028f4c9508bd.png)
![image](https://user-images.githubusercontent.com/1778417/176918280-0efdd940-d5d9-45b0-a8ac-c893ec240b16.png)

**Comment by Zolyn** [comment]
Could you give a reproduction link?

**Comment by Matodor** [comment]
> Could you give a reproduction link?

Yes, in a few hours I will try to make it available to the public.

**Comment by Matodor** [comment]
Oh, I had some exports from the "<script setup>" section. 
They are not allowed by "vue/compiler-sfc". 
After removing them, the error disappeared. But the props still don't bind.

I read "Known limitations" from the repository "@zolyn/vite-plugin-vue-type-imports":
* `nested type parameters (such as defineProps<Props<T>>>()) are not supported.`

This seems to be my case 😁
[Codesandbox](https://codesandbox.io/s/silent-rain-mgdjbp?file=/src/components/Button.vue)

**Comment by Matodor** [comment]
I was able to reproduce the error.
![image](https://user-images.githubusercontent.com/1778417/176966865-bf55bdc2-2674-4b4e-ba1d-97cf67c26160.png)

https://codesandbox.io/s/floral-fire-mwsg65
